frontend/packages/dev-console/src/components/import/git/GitSection.tsx

```diff
@@ -95,6 +79,40 @@ const GitSection: React.FC<GitSectionProps> = ({ showSample, builderImages }) =>
     ],
   );
 
+  const handleBuilderImageRecommendation = React.useCallback(async () => {
+    const gitType = gitTypeTouched ? values.git.type : detectGitType(values.git.url);
+    const gitService = getGitService(
+      { url: values.git.url, ref: values.git.ref },
+      gitType as GitProvider,
+    );
+    if (isRepoReachable && builderImages) {
+      setFieldValue('image.isRecommending', true);
+      const buildTools: BuildType[] = await gitService.detectBuildTypes();
```

Comment:
Encountered this error with the following steps:

- go to Add => From Git
- Enter a valid git url and wait for validation to complete
- delete the git url

![image](https://user-images.githubusercontent.com/14068621/98027153-3689ab00-1dda-11eb-8d8d-fbb9ef26a288.png)